tools/pythonpkg/tests/fast/pandas/test_timedelta.py

```diff
@@ -1,10 +1,16 @@
+import platform
 import pandas as pd
 import duckdb
 import datetime
-import numpy as np
 import pytest
 
 
+pytestmark = pytest.mark.skipif(
+    platform.system() == "Emscripten",
+    reason="Not supported on Emscripten",
+)
```

Comment:
This is skipped even though technically it shouldn't be. Without the skipping this actually crashes the interpreter with a funky WASM stack trace about `BigInt`. I can dig around in a follow-up.

Reply:
Sure, this is a clear improvement on current status, tests are run and failure are mapped.